demo/cluster/aws-standalone-istio-child.yaml

```
@@ -0,0 +1,22 @@
+apiVersion: k0rdent.mirantis.com/v1alpha1
+kind: ClusterDeployment
+metadata:
+  name: aws-ue2-child-istio
```

For consistency with the filename, and it sounds better:
```suggestion
  name: aws-ue2-istio-child
```